knimin/lib/tests/test_mail.py

```diff
@@ -0,0 +1,32 @@
+from unittest import TestCase, main
+
+from knimin.lib.mail import send_email
+
+
+class TestEmail(TestCase):
+    def test_send_email_basic(self):
```

Comment:
No, it just gets the email MIME object and checks it's set up correctly. It does not actually send the email, as we can't be sure an email server is set up.